GameOfRunes/GameOfRunes/Custom Nodes/Player Area/SummonNode.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  SummonNode.swift
+//  GameOfRunes
+//
+//  Created by Jermy on 17/3/20.
+//  Copyright © 2020 TeamHoWan. All rights reserved.
+//
+
+import SpriteKit
+
+class SummonNode: SKSpriteNode {
```

Comment:
Just to clarify, this is the node representing the button we tap on to summon an enemy right. So this node basically functions as a button?

Reply:
Yep.